vulnerable_people_form/integrations/location_eligibility.py

```diff
@@ -42,7 +42,7 @@ def get_uprn_tier(uprn):
     if not records:
         logger.info(create_log_message(
             log_event_names["UPRN_TIER_NOT_FOUND"],
-            f"No tier found in the database for uprn: {uprn}"
+            f"No tier found in the database for uprn: <redacted>"
```

Comment:
the f strings are not needed in this case as nothing is being replaced - this should also fix the linting issue